commit f5fe47571f00aad2acd291e6f2ed2d6b4e543288
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Apr 27 16:21:33 2018 +0100

    config: Add I2C_TINY_USB=m
    
    Enable the I2C Tiny USB module.
    
    See: https://github.com/raspberrypi/linux/issues/2535
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 143ee2ebc7e2..bc26764f6f70 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -610,6 +610,7 @@ CONFIG_I2C_BCM2708=m
 CONFIG_I2C_BCM2835=m
 CONFIG_I2C_GPIO=m
 CONFIG_I2C_ROBOTFUZZ_OSIF=m
+CONFIG_I2C_TINY_USB=m
 CONFIG_SPI=y
 CONFIG_SPI_BCM2835=m
 CONFIG_SPI_BCM2835AUX=m
@@ -905,7 +906,6 @@ CONFIG_SND_SOC_CS4271_I2C=m
 CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_WM8804_I2C=m
 CONFIG_SND_SIMPLE_CARD=m
-CONFIG_SND_AUDIO_GRAPH_CARD=m
 CONFIG_HID_BATTERY_STRENGTH=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 73e5ffa2625c..34f1b22c76ff 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -605,6 +605,7 @@ CONFIG_I2C_BCM2708=m
 CONFIG_I2C_BCM2835=m
 CONFIG_I2C_GPIO=m
 CONFIG_I2C_ROBOTFUZZ_OSIF=m
+CONFIG_I2C_TINY_USB=m
 CONFIG_SPI=y
 CONFIG_SPI_BCM2835=m
 CONFIG_SPI_BCM2835AUX=m
@@ -898,7 +899,6 @@ CONFIG_SND_SOC_CS4271_I2C=m
 CONFIG_SND_SOC_SPDIF=m
 CONFIG_SND_SOC_WM8804_I2C=m
 CONFIG_SND_SIMPLE_CARD=m
-CONFIG_SND_AUDIO_GRAPH_CARD=m
 CONFIG_HID_BATTERY_STRENGTH=y
 CONFIG_HIDRAW=y
 CONFIG_UHID=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1239_f5fe47571f00aad2acd291e6f2ed2d6b4e543288.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1239_f5fe47571f00aad2acd291e6f2ed2d6b4e543288.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
